lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -19,7 +19,7 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
   /// [domElement] so the caller can insert it before calling
   /// [SemanticsTextEditingStrategy.enable].
   SemanticsTextEditingStrategy(
-      HybridTextEditing owner, html.HtmlElement domElement)
+      HybridTextEditing owner, html.Element domElement)
```

Comment:
Can you change to HtmlElement? dart:html will start checking for if (element is SvgElement) all over the place since html.Element is either svg or htmlelement.

Reply:
Done.